batavia/types/Str.js

```diff
@@ -654,6 +654,14 @@ Str.prototype.__hash__ = function() {
     return new types.Int(h)
 }
 
+Str.prototype.capitalize = function() {
+    if(arguments.length === 0){
```

Comment:
recently i was writing mostly on c#, so i have used the same style here. Thank you!